TeamMandate/Presentation.md

```diff
@@ -10,112 +10,80 @@ As a student who isn't taking any courses this summer, I decided to look for wor
 
 ... 
 
-## The How
+## Proposed IITB Mandated Priorities
 
-- Do not compete with private sector 
-  - Integration over development 
-  - Create jobs, not software 
+### Avoid Rework 
 
->delivering programs that help Canadians move through life’s transitions, from school to work, from one job to another, *from unemployment to employment* and from the workforce to retirement; 
+Work in the open
 
-- Avoid rework 
-  - Work in the open 
+#### Repercussions
+
+- All teams (Ex: developers, security and testing) must use git
+
+#### Opportunities
+
+- Visibility will allow teams to understand an always up to date version of the processes of other teams 
+- Reduce rework from sharing and building on the work of others 
+  - Example: CDS implements a secure document uploading project. CSE 
 
 >Values and Ethics Code for the Public Sector: 
 Federal public servants are entrusted to use and care for public resources responsibly, for both the short term and long term.
 
-- Modern tools 
-  - Fully separated client and back ends 
-  - All communications through service interfaces
-  - Communications between teams must be through these interfaces
+... 
+
+### Modern tools 
```

Comment:
- Support bring your own device. Un-managed devices not connected to ESDC's network but able to securely access work in progress